core/src/main/scala/kafka/server/DynamicConfigManager.scala

```diff
@@ -154,7 +156,7 @@ class DynamicConfigManager(private val zkClient: KafkaZkClient,
   }
 
   private val configChangeListener = new ZkNodeChangeNotificationListener(zkClient, ConfigEntityChangeNotificationZNode.path,
-    ConfigEntityChangeNotificationSequenceZNode.SequenceNumberPrefix, ConfigChangedNotificationHandler)
+    ConfigEntityChangeNotificationSequenceZNode.SequenceNumberPrefix, ConfigChangedNotificationHandler, changeExpirationMs, time)
```

Comment:
We should pass the `changeExpirationMs` and `time` into `ZkNodeChangeNotificationListener`, otherwise, it'll always use default value.

Reply:
You're right. Will do. Thanks.

Reply:
Also, shall we change the code so that we are forced to pass this?

Reply:
So far, we only use dynamicConficManager during broker started. So, we can keep as before and use default one for sure. But pass there's parameters make is more flexible is not bad. I don't have any preference here. What do you think?
Thank you.

Reply:
Default arguments are dangerous for things like this, I would remove them.

Reply:
Agree! Removed. Thanks.